pytorch/buildspec-graviton-1.12.yml

```diff
@@ -1,9 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK pytorch
-version: &VERSION 1.10.0
```

Comment:
Please revert these buildspec changes. We track the "latest" framework version in the un-versioned buildspecs, and older framework versions in versioned buildspecs.

If you would like to build older graviton images in PR context, please set graviton_mode to True in the developer toml, and also fill in the path to the appropriate buildspec in the buildspec_overrides section (https://github.com/aws/deep-learning-containers/blob/master/dlc_developer_config.toml#L95-L96)

Reply:
reverted and properly done. Thank you.